blog/2021-03-22-weekly-report.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 2021-11 Weekly Report
+author: Xuanwo
+author_url: https://github.com/Xuanwo
+tags: [updates, weekly-report]
+---
+
+This the weekly report for [AOS](https://aos.dev), range from 2021-03-15 to 2021-03-21.
+
+## Application
+
+### dm
+
+This week we implemented the data flow related logic and next week we plan to implement the upper level business logic so that users can create and run a migration task.
+
+## Library
+
+### go-storage
+
+This week we released: [v3.4.2](https://github.com/aos-dev/go-storage/releases/tag/v3.4.2)
+
+In this release we implement multipart related fields like:
+
+- `MultipartNumberMaximum`
+- `MultipartSizeMaximum`
+- `MultipartSizeMinimum`
+
+So we can get service restrictions on multipart.
+
+## Community
+
+We set up following sites:
+
+- AOS's blog: <https://aos.dev/blog>
+- dm's Wiki: <https://github.com/aos-dev/dm/wiki>
+
+We had a meeting of roadmap of 2021 Q2, slides: <https://docs.google.com/presentation/d/1jC30T4S-eFrtIUoj9QStv8UDXq65UJaJmP0R2y-HqXo/>
```

Comment:
Use md url like `[slides](http:foo.bar)` instead?

Reply:
I think current style is good enough.